flake.nix

```diff
@@ -6,7 +6,7 @@ rec {
 
   # These inputs are used for the manuals and release artifacts
   inputs.released-nixpkgs-unstable = { url = "nixpkgs/nixos-unstable"; };
-  inputs.released-nixpkgs-stable = { url = "nixpkgs/nixos-22.05"; };
+  inputs.released-nixpkgs-stable = { url = "nixpkgs/release-22.11"; };
```

Comment:
Shouldn't this use `nixos-22.11` as it did for 22.05?

Reply:
That's just temporary difference, so that we don't need to wait (probably for a few hours).

Reply:
This is a temporary hack since we'd like to get the release out today and nixos-22.11 has not been built since the release commit (and getting a successful nixos-22.11 bump will take many hours).

Reply:
Ninja'd ;)

Thanks.

Reply:
It should, be we don't have an evaluation that has advanced so far as to include the final release notes changes. So we're going for release-22.11 but will fallback to nixos-22.11 when it has caught up.

Reply:
At this point we might already use `nixos-22.11` already, so why not?  We ended up having to wait for it anyway, due to the issue with aarch64 ISO redirects.